docs/codeql/query-help/readme.txt

```diff
@@ -0,0 +1,8 @@
+CodeQL query help Sphinx documentation
```

Comment:
Minor: Can this README be a `.md` instead of `.txt` file? Just to get the nice formatting 🙂

Reply:
👍🏻

Reply:
Thanks! And good point about excluding it from the build - I hadn't thought about that 👏🏽